### ISSUE TITLE
Doesn't post

### ISSUE THREAD

**Issue by anon17** [CyberShadow/DFeed#116]
Posting zkmrhvjcivyptoqndgwd can't pass for some days.

**Comment by CyberShadow** [comment]
- Looking at the logs, I see that it was submitted on 2019-10-29 09:43:36, and successfully posted at with the message ID ebufcpdbifumalhwakpv@forum.dlang.org (NNTP server said `240 Article received <ebufcpdbifumalhwakpv@forum.dlang.org>`). However, https://forum.dlang.org/post/ebufcpdbifumalhwakpv@forum.dlang.org does not exist, so the NNTP server apparently decided to discard or otherwise failed to save the message.

- Upon opening https://forum.dlang.org/posting/zkmrhvjcivyptoqndgwd , I clicked "Send" and it went through. It was posted as https://forum.dlang.org/post/egiczeoryyryiliyhnzp@forum.dlang.org.

- I noticed that the server continued to display a form at https://forum.dlang.org/posting/zkmrhvjcivyptoqndgwd even after I clicked send. I've fixed this now in 43dd4d7b5227ffb34c6bc6ff2aa7dc15c23433eb.
